os/customizing-sshd.md

```diff
@@ -23,11 +23,11 @@ storage:
           # Use most defaults for sshd configuration.
           UsePrivilegeSeparation sandbox
           Subsystem sftp internal-sftp
+          UseDNS no
 
           PermitRootLogin no
           AllowUsers core
-          PasswordAuthentication no
-          ChallengeResponseAuthentication no
```

Comment:
My preference comes from Mozilla's config (https://wiki.mozilla.org/Security/Guidelines/OpenSSH#Modern_.28OpenSSH_6.7.2B.29) which is at least a form of documentation.

Based on the comments in that, it looks like these options are older and thus more widely supported. Fortunately, we don't have to support openssh 5.3